src/ocsp.c

```diff
@@ -582,23 +599,37 @@ WOLFSSL_OCSP_CERTID* wolfSSL_OCSP_cert_to_id(
 
     certId = (WOLFSSL_OCSP_CERTID*)XMALLOC(sizeof(WOLFSSL_OCSP_CERTID), NULL,
                                            DYNAMIC_TYPE_OPENSSL);
```

Comment:
In OpenSSL compatibility functions, should we allocate all types with DYNAMIC_TYPE_OPENSSL? What are the guidelines around setting the allocation type?

Reply:
Main requirement is to use the same type on XMALLOC and XFREE. Use best judgement on available types. If merits add a new dynamic type.